src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -416,6 +427,31 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
           </Box>
         ))}
       </Modal>
+
+      <Modal
```

Comment:
hadde nesten glemt at vi har vår egen Modal-komponent :D den skulle vi kanskje brukt mer faktisk, for at alle skal få kryss opp til høyre etc